ts/handlers/html/HTMLDocument.ts

```diff
@@ -349,6 +349,9 @@ export class HTMLDocument<N, T, D> extends AbstractMathDocument<N, T, D> {
    */
   public addStyles(styles: StyleJson) {
     this.styles.push(styles);
+    if ('insertStyles' in this.outputJax) {
```

Comment:
Would this be better?
```suggestion
    if (Object.hasOwn(this.outputJax, 'insertStyles') {
```

Reply:
This won't work, because 'inserStyles` is in the prototype, not a direct property of the object.  That means that `Object.hasOwn()` will not find it, but `in` will.